### ISSUE TITLE
Timezone awareness with MomentJS

### ISSUE THREAD

**Issue by mrfhitz** [mrfhitz/Rocket-Launch-Schedule#5]
Use [MomentJS](https://momentjs.com/timezone/) to be sure all the timezone's are covered. 

_I need to think better how am i gonna do this._

**Comment by Nosudrum** [comment]
You could perhaps also use the date/time format from the client's settings

**Comment by mariogarridopt** [comment]
Yes, there is some work to be done [here](https://github.com/mariogarridopt/Rocket-Launch-Schedule/blob/master/js/datetime.js).
I would prefer to use native js stuff than a lib.
